test/helpers/cons.go

```diff
@@ -192,17 +192,11 @@ const (
 	IPv6Host = "fdff::ff"
 
 	// Logs messages that should not be in the cilium logs.
-	panicMessage        = "panic:"
-	deadLockHeader      = "POTENTIAL DEADLOCK:"               // from github.com/sasha-s/go-deadlock/deadlock.go:header
-	segmentationFault   = "segmentation fault"                // from https://github.com/cilium/cilium/issues/3233
-	NACKreceived        = "NACK received for version"         // from https://github.com/cilium/cilium/issues/4003
-	RunInitFailed       = "JoinEP: "                          // from https://github.com/cilium/cilium/pull/5052
-	selfishThresholdMsg = "Goroutine took lock for more than" // from https://github.com/cilium/cilium/pull/5268
```

Comment:
Do we need to drop this to implement this feature? I think it's important to keep this visible.

Reply:
Oh, it just got moved to the other file, never mind:)